packages/core-mobile/app/store/rpc/providers/walletConnect/utils.ts

```diff
@@ -9,5 +8,15 @@ export const isSessionProposal = (
   return request.method === RpcMethod.WC_SESSION_REQUEST
 }
 
-export const isUserRejectedError = (error: RpcError): boolean =>
-  error.code === providerErrors.userRejectedRequest().code
+export const isUserRejectedError = (error: unknown): boolean => {
```

Comment:
would it be possible to make typing bit stronger here?
By looking just at function signature one would think it could be used anywhere, but it's actually specific for metamask/rpc-errors group of errors, and code number could clash with some other group of errors, right?

Reply:
it's actually being used everywhere right now (in both react component and redux) so that's why I adjusted it to unknown. the function now also checks for the correct type or else it will return false.